third_party/terraform/utils/config.go.erb

```diff
@@ -190,6 +190,7 @@ var <%= product[:definitions].name -%>DefaultBasePath = "<%= product[:definition
 var defaultClientScopes = []string{
 	"https://www.googleapis.com/auth/compute",
 	"https://www.googleapis.com/auth/cloud-platform",
+	"https://www.googleapis.com/auth/cloud-identity",
```

Comment:
It's probably worth adding a note to the changelog about this (we did last time: https://github.com/GoogleCloudPlatform/magic-modules/pull/2473)